src/graphics.c

```diff
@@ -641,62 +672,72 @@ void graphics_draw_character( display_context_t disp, int x, int y, char ch )
 
     int depth = __bitdepth;
 
+    // setting default font if none was set previously
+    if( sprite_font.sprite == NULL )
+    {
+        sprite_font.sprite = (sprite_t *)(depth == 2 ? __font_data_16 : __font_data_32);
```

Comment:
Shouldn't this configure the whole `sprite_font` structure for the system font? Including `font_width`, `font_height` and `tab_width`. Maybe you could just call `graphics_set_font` here.

Reply:
Also, I think we should possibly export the system font to the public API now. So maybe having a system_font.c file with its header file. That's because otherwise the user has no way to explicitly select the system font after selecting something else.

Reply:
Given we have different font according to the bits used, wouldn't it be better to just have a function (eg. `graphics_set_default_font`) to set the default font?

Reply:
Ok that's fine as well